src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/Read/ClassRoomItem.tsx

```diff
@@ -1,34 +1,71 @@
-import { Text, Box } from 'grommet';
+import { Text, Box, CheckBox } from 'grommet';
 import { FormSchedule, InProgress } from 'grommet-icons';
 import {
   ClassroomLite,
   StyledLink,
   ContentCard,
   TextTruncated,
 } from 'lib-components';
-import { Fragment } from 'react';
+import { Fragment, useEffect, useState } from 'react';
 import { useIntl } from 'react-intl';
 
 import { ReactComponent as VueListIcon } from 'assets/svg/iko_vuelistesvg.svg';
 import { ReactComponent as ClassroomsIcon } from 'assets/svg/iko_webinairesvg.svg';
+import { useSelectFeatures } from 'features/Contents/store/selectionStore';
 import { routes } from 'routes';
 import { localDate } from 'utils/date';
 
 const ClassRoom = ({ classroom }: { classroom: ClassroomLite }) => {
   const intl = useIntl();
   const classroomPath = routes.CONTENTS.subRoutes.CLASSROOM.path;
 
+  const { isSelectionEnabled, selectedItems, selectItem } = useSelectFeatures();
+  const [isClassroomSelected, setIsClassroomSelected] = useState<boolean>(
+    () => selectedItems.includes(classroom.id) || false,
+  );
+
+  useEffect(() => {
+    if (!isSelectionEnabled) {
+      setIsClassroomSelected(false);
+    }
+  }, [isSelectionEnabled, setIsClassroomSelected]);
+
   return (
-    <StyledLink to={`${classroomPath}/${classroom.id}`}>
+    <StyledLink
+      to={isSelectionEnabled ? '#' : `${classroomPath}/${classroom.id}`}
```

Comment:
The problem with the scrollUp comes from here, the click on the item mutate `location` inside AppRoutes.
The solution is to be more specific in our dependencies:
Change to `location.pathname` [here](https://github.com/openfun/marsha/blob/master/src/frontend/apps/standalone_site/src/routes/AppRoutes.tsx#L50).